lib/Dialect/Torch/IR/TorchOps.cpp

```diff
@@ -3979,6 +3979,18 @@ OpFoldResult AtenAddFloatIntOp::fold(FoldAdaptor adaptor) {
       adaptor.getOperands(), [](double a, double b) { return a + b; });
 }
 
+//===----------------------------------------------------------------------===//
+// AtenMulFloatIntOp
+//===----------------------------------------------------------------------===//
+
+OpFoldResult AtenMulFloatIntOp::fold(FoldAdaptor adaptor) {
+  if (!adaptor.getA() || !adaptor.getB()) {
+    return nullptr;
+  }
+  return atenBinaryFloatOperatorFoldHelper(
+      adaptor.getOperands(), [](double a, double b) { return a * b; });
```

Comment:
Similar to my comment in your other PR, one of the two arguments should be an `int64_t`, and it would be beneficial to have additional folding options for whenever one of the two operands is `1` or `0`.

Reply:
See other PR.

Reply:
```suggestion
      adaptor.getOperands(), [](double a, int64_t b) { return a * static_cast<double>(b); });
```

Reply:
Nvm, I just read the discussion on the other PR.